easybuild/tools/modules.py

```diff
@@ -1415,6 +1417,28 @@ def update(self):
                     mkdir(cache_dir, parents=True)
                 write_file(cache_fp, stdout)
 
+    def _set_module_path(self, new_mod_path):
+        """
+        Set $MODULEPATH to the specified paths and logs the change.
+        new_mod_path can be None or an iterable of paths
+        """
+        if new_mod_path is not None:
+            if not isinstance(new_mod_path, list):
+                assert not isinstance(new_mod_path, string_type)  # Just to be sure
+                new_mod_path = list(new_mod_path)  # Expand generators
+            new_mod_path = mk_module_path(new_mod_path) if new_mod_path else None
+        cur_mod_path = os.environ.get('MODULEPATH')
+        if new_mod_path != cur_mod_path:
+            self.log.debug(
```

Comment:
This is worth `log.info` perhaps?

Reply:
This matches behavior of current `module use/unuse` which does a debug log already. That's ok IMO, as all this changing of variables back and forth is not something you need to know for a normal, successful installation but only when debugging an issue.